src/sentry/snuba/metrics/query_builder.py

```diff
@@ -186,35 +200,11 @@ def parse_query(query_string: str, projects: Sequence[Project]) -> Sequence[Cond
         query_builder = UnresolvedQuery(
             Dataset.Sessions,
             params={
-                "project_id": 0,
+                "project_id": [project.id for project in projects],
```

Comment:
is it okey that we now pass instead of single `project id` list of project_ids?

Reply:
Yeah, it was broken before. This actually fixes it